passive/Server Header Disclosure.js

```diff
@@ -15,9 +16,27 @@ function scan(ps, msg, src)
     var url = msg.getRequestHeader().getURI().toString();
     var headers = msg.getResponseHeader().getHeaders("Server")
     
-    if (headers != null)
+    if (headers != null && containsPotentialServer(headers))
     {
-        ps.raiseAlert(alertRisk, alertConfidence, alertTitle, alertDesc, url, '', '', '', alertSolution,headers, cweId, wascId, msg);
+        var headers_string = headers.toString();
+        ps.raiseAlert(alertRisk, alertConfidence, alertTitle, alertDesc, url, '', '', '', alertSolution, headers_string, cweId, wascId, msg);
     }
     
 }
+
+function containsPotentialServer(content) {
```

Comment:
`containsPotentialSemver` I didn't notice this earlier, sorry about that.

I had meant for it to purposefully be "semver" meaning short form for 'Semantic Version' (ex: https://semver.org)

Reply:
Should be as intended now.